### Strange formating at the end of the file
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:functions:describe-object.txt	2017-04-27 21:59:29.000000000 +0100
+++ saved-pages-clus/cl:functions:describe-object.txt	2017-04-27 21:59:29.000000000 +0100
@@ -3,1 +4,1 @@
 Syntax


-describe-object object stream => implementation-dependent
+ describe-object object stream → implementation-dependent

 Method Signatures

@@ -23,38 +42,88 @@
  Examples


  (defclass spaceship ()
-   ((captain initarg captain accessor spaceship-captain)
-    (serial# initarg serial-number accessor spaceship-serial-number)))
+  ((captain initarg captain
+            accessor spaceship-captain)
+   (serial-number initarg serial-number
+                  accessor spaceship-serial-number)))
+→
+SPACESHIP

  (defclass federation-starship (spaceship) ())

+→
+FEDERATION-STARSHIP
  (defmethod describe-object ((s spaceship) stream)
-   (with-slots (captain serial#) s
-     (format stream "~&~S is a spaceship of type ~S,~
-                     ~%with ~A at the helm ~
-                       and with serial number ~D.~%"
-             s (type-of s) captain serial#)))
+  (with-slots (captain serial-number) s
+    (format stream "~&~S is a spaceship of type ~S,~%~
+            with ~A at the helm and with serial number ~D.~%"
+            s (type-of s) captain serial-number)))

- (make-instance 'federation-starship
+→
+#<STANDARD-METHOD COMMON-LISPDESCRIBE-OBJECT (FOO T) {10026FD5D3}>
+([CLFunctionsmake-instance]] 'federation-starship
                 captain "Rachel Garrett"
                 serial-number "NCC-1701-C")
-=>  #<FEDERATION-STARSHIP 26312465>
+→
+#<FEDERATION-STARSHIP 26312465>

  (describe *)
->>  #<FEDERATION-STARSHIP 26312465> is a spaceship of type FEDERATION-STARSHIP,
->>  with Rachel Garrett at the helm and with serial number NCC-1701-C.
-=>  <no values>

-<TT>
-Affected By None.
+▷
+#<FEDERATION-STARSHIP 26312465> is a spaceship of type FEDERATION-STARSHIP,
+with Rachel Garrett at the helm and with serial number NCC-1701-C.
+→
+<no values>
+</blockquote>

-Exceptional Situations None.

-See Also

-describe
-Notes
+====Affected By====
+None.

+====Exceptional Situations====
+None.
+====See Also====
+Function DESCRIBE
+====Notes====
 The same implementation techniques that are applicable to print-object are applicable to describe-object.
 The reason for making the return values for describe-object unspecified is to avoid forcing users to include explicit (values) in all of their methods. describe takes care of that.
\ No newline at end of file
+\
\ No newline at end of file
